src/components/Autocomplete/README.md

```diff
@@ -55,21 +55,22 @@ class AutocompleteExample extends React.Component {
   };
 
   render() {
+    const textField = (
+      <Autocomplete.TextField
+        onChange={this.updateText}
+        label=""
```

Comment:
I think we should add labels to the example

Reply:
> We don't use labels for any of the search fields in production

I wonder why? We could probably use something as basic as `label="search"`

Reply:
We use placeholders.

![screen shot 2018-11-19 at 7 27 53 am](https://user-images.githubusercontent.com/6239480/48716658-a8a45d80-ebcc-11e8-8ae9-59b60a2812bf.jpg)